bin/test_client.rb

```diff
@@ -120,6 +122,18 @@ def test_send_letter_endpoint(client)
   letter_resp
 end
 
+def test_send_precompiled_letter_endpoint(client)
+  pdf_file = File.open('spec/test_files/test_pdf.pdf', 'rb')
```

Comment:
This doesn't matter as much in this test, but `File.open` has a block syntax which is quite similar to python's `with open` and makes sure the file is closed automatically:

```ruby

File.open('spec/test_files/test_pdf.pdf', 'rb') { |pdf_file|
    ...
}
```